.github/workflows/build.yaml

```
@@ -0,0 +1,144 @@
+name: Build, tag, and publish
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - develop
+    paths:
+      - '!*'
+      - 'src/**'
+      - 'package.json'
+      - 'cypress.json'
+      - 'cypress/**'
+      - 'Dockerfile'
+      - '.github/workflows/dev-image-update.yaml'
```

This file no longer exists. It seems like it should refer to itself again, as it did before. Or maybe it should include all github workflows?
```suggestion
      - '.github/workflows/build.yaml'
```

---

Thanks, fixed.